ui/src/message_popup/playtest/hmtca_group_review.jsx

```diff
@@ -0,0 +1,103 @@
+/* @flow weak */
+import React from 'react';
+import _ from 'lodash';
+
+import * as Api from '../../helpers/api.js';
+import Paper from 'material-ui/Paper';
+import {Card, CardTitle, CardText} from 'material-ui/Card';
+
+
+// For reviewing responses as a group.
+export default React.createClass({
+  displayName: 'HMTCAGroupReview',
+
+  propTypes: {
+    applesKey: React.PropTypes.string
+  },
+
+  getInitialState() {
+    return {
+      hasLoaded: false,
+      applesResponses: null
+    };
+  },
+
+  componentDidMount() {
+    this.refreshResponses();
+  },
+
+  refreshResponses() {
+    const {applesKey} = this.props;
+    Api.getApples(applesKey).end(this.onResponsesReceived);
+  },
+
+  onResponsesReceived(err, response){
+    if (err){
+      this.setState({ hasLoaded: true });
+      return;
+    }
+    const applesResponses = JSON.parse(response.text);
+    this.setState({ hasLoaded: true, applesResponses });
+  },
+
+  render() {
+    const {hasLoaded, applesResponses} = this.state;
+    if (!hasLoaded) return <div>Loading...</div>;
+
+    // Use scene number for ordering, but then just number then naturally
+    const sceneToResponses = _.groupBy(applesResponses, 'scene_number');
```

Comment:
This sorts out the responses by scene, and then goes through each scene in order, and then makes a sticky-note-looking card for each anonymized response.